src/model/transaction/Deadline.ts

```diff
@@ -25,7 +25,7 @@ export class Deadline {
     /**
      * @type {number}
      */
-    public static timestampNemesisBlock = 1573430400;
+    private nemesisEpoch: number;
```

Comment:
Hi Steve, I've swapped the internal fields of the Deadline object in java. 

Instead of keeping both values (local and epoch) you can store the server value as-is.  When serializing and deserializing from and back dto and catbuffer you don't need to provide the epochAdjustment (the mappers won't need access to a configuration).  

epochAdjustment is only necessary when users creates a relative deadline from current timestamp (like now + 2 hours).

https://github.com/nemtech/symbol-sdk-java/blob/351_configurable_epoch_time/sdk-core/src/main/java/io/nem/symbol/sdk/model/transaction/Deadline.java#L35

This change would not require a client change.

Reply:
`public value: LocalDateTime;` would not apply anymore. 

we could have a 

public toLocalDateTime(epochAdjustment:number):LocalDateTime that converts the as-is server value back to a real "date"